patternfly/patternfly-elements
# fix: Navigation styles for mobile accordion

## Navigation mobile styles

Expected:
![image](https://user-images.githubusercontent.com/1840295/107795462-acc97200-6d26-11eb-9397-640dcf3da31b.png)

On main branch:
![image](https://user-images.githubusercontent.com/1840295/107795489-b4891680-6d26-11eb-90bd-8577236679b8.png)


### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1353--happy-galileo-ea79c4.netlify.app/elements/pfe-navigation/demo//) 


### What has changed and why

- Sass typo: `$accordion: pfe-accordion-variables;` needed to be `$accordion: pfe-accordion-variables();`
- Add IE11-specific styles to make the language & login links readable
- Add mobile snapshot for navigation
- 

### Testing instructions

1. Open the [demo page](https://deploy-preview-1353--happy-galileo-ea79c4.netlify.app/elements/pfe-navigation/demo/)
  - Navigation should upgrade
2. Resize to less than 500px wide
3. Click Menu
  - Navigation items in the menu dropdown should be styled like accordions.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
